tests/testthat/test-stat_stepribbon.R

```diff
@@ -0,0 +1,130 @@
+#' @title Specifications test-stat_stepribbon.R
+#' @section Last updated by: Daniel Sjoberg (danield.sjoberg@@gmail.com)
+#' @section Last update date: 2022-07-12T04:47:32
+#'
+#' @section List of tested specifications
+#' T1. visR::stat_stepribbon() accepts aes() args ymin and ymax.
+#' T1.1 visR::stat_stepribbon() inherits aes from ggplot()
+#' T1.2 visR::stat_stepribbon() respects aes from visR::stat_stepribbon()
+#' T2. visR::stat_stepribbon() works with the color/fill aesthetic.
+#' T2.1 The step ribbon is correctly applied when there are multiple lines from the color/fill aesthetic
```

Comment:
define correctly

Reply:
please update as you see fit

Reply:
I still dont know what you mean with this test. If I understand, I dont mind updating.

Reply:
you can update the text or even the unit tests if youd like

Reply:
multiple lines = stratified KM curve (this however is a general function and does nor strictly apply to KM curves)